README.rst

```diff
@@ -42,6 +43,20 @@ This can be used then with a command like this::
 
     source ~/Downloads/mc3/bin/activate
 
+Windows Example:
+
+On Windows if you run like this::
+
+    python3 mcinstall.py %USERPROFILE%\downloads\mc3
```

Comment:
This refers to using the script inside the repo without pip-installing it. Can we either make this clear or align with the other examples which use the pip-installed tool named `mcinstall` without extension?

Reply:
sure will change this